ksql-parser/src/test/java/io/confluent/ksql/parser/SqlFormatterTest.java

```diff
@@ -174,6 +180,23 @@ public void setUp() {
     metaStore.putSource(ksqlTableOrders);
   }
 
+  @Test
+  public void shouldFormatCreateStreamStatementWithExplicitKey() {
+    // Given:
+    final CreateStream createStream = new CreateStream(
+        QualifiedName.of("TEST"),
+        ELEMENTS_WITH_KEY,
+        false,
+        SOME_WITH_PROPS);
+
+    // When:
+    final String sql = SqlFormatter.formatSql(createStream);
+
+    // Then:
+    assertThat(sql, is("CREATE STREAM TEST (ROWKEY STRING KEY, Foo STRING) "
+        + "WITH (VALUE_FORMAT='JSON', KAFKA_TOPIC='topic_test', KEY='ORDERID');"));
```

Comment:
Remove `ORDERID`as KEY. We don't have it as a column in the schema.

Reply:
You're right, in that this would fail to execute, but the formatter doesn't care about this. Nor should it.  It's just formatting what it's passed.  If I remove `KEY` here the test would fail.